server/schedule/placement/rule_manager.go

```diff
@@ -268,7 +268,7 @@ func (m *RuleManager) GetRulesForApplyRegion(region *core.RegionInfo) []*Rule {
 		// or
 		//   |<-- rule -->|
 		// |<--- region --->|
-		return nil
+		return nil // It will considered abnormal when no rules for the regoin. Then Rule checker will try to split the region.
```

Comment:
"It will considered abnormal when a region is not covered by any rule." seems better.

Reply:
Good suggestion. Thanks!